### DISCUSSION TITLE
Make crud routes name unique with some name prefix

### DISCUSSION THREAD

**Discussion by lotarbo** [Laravel-Backpack/community-forum#289] [Ideas]
What i mean:
I have page crud, List operation register route with name **page.index**
I have page controller and web route '/' with name **page.index**

Yep, i can rename my route, but i think, it would be nice, if backpack can add prefix name via config, ex: **crud.page.index** name for list route)
@tabacitu what you think?

**Comment by tabacitu** [comment]
Yup! I think removing the `admin` prefix in 4.0 wasn't such a good idea. Let's move the conversation to https://github.com/Laravel-Backpack/ideas/issues/2 please (more people there).

In the meantime, you can [add a name prefix to your custom routes right now](https://github.com/Laravel-Backpack/ideas/issues/2#issuecomment-538276755). The only problem here is... your _custom_ routes will have this prefix. But CRUDs that come as addons (third-party packages) won't.

We plan to fix this problem in Backpack 4.2, which is right around the corner.
